app/api/v1/endpoints/trading.py

```
@@ -559,40 +591,46 @@ async def get_recent_trades(
-        ]
-        return RecentTradesResponse(recent_trades=mock_recent_trades)
+        # Get real recent trades from database
+        from sqlalchemy import select, desc
+        from app.models.trading import Trade, TradeAction
+        
+        # Query recent trades for user
+        stmt = select(Trade).where(
+            Trade.user_id == current_user.id
+        ).order_by(desc(Trade.created_at)).limit(50)
+        
+        result = await db.execute(stmt)
```

Ruff (0.12.2)

Undefined name `db`

(F821)